### ISSUE TITLE
Suggestion: Support keyboard navigation for all interactive elements

### ISSUE THREAD

**Issue by fpbrault** [script3/yieldblox-beta#335]
Only part of the UI elements can be interacted with using a keyboard. For example, while it is possible to focus the lend/borrow buttons, it is not possible to focus the assets themselves.

This pretty much prevents any use of the app for users that are unable to use a mouse, such as people with motor function impairment, as they cannot lend or borrow assets.

Additionally, it is not possible to claim YBX, or to disconnect the wallet using only a keyboard.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard
